### DISCUSSION TITLE
Can’t access desktop version of website on mobile

### DISCUSSION THREAD

**Discussion by getaaron** [community/community#48305] [Other Feature Feedback, Questions, & Ideas]
On my iPhone I want to download a zip file of a repository. I used to be able to do this by requesting the desktop website and then clicking download zip. However, the request desktop site option no longer appears on the GitHub mobile website. Additionally, Safari's "request desktop site" function has no effect.

**Comment by ericoporto** [comment]
I have the same issue, no way to get the desktop website on mobile Safari, and the iOS GitHub App is too terrible and lacks anything useful.

Just give me the desktop website on iPhone.
